.github/workflows/npm-publish.yml

```diff
@@ -7,20 +7,23 @@ on:
   push:
     branches:
       - main
+      - beta
+      - alpha
 
 jobs:
   semantic-release:
     name: semantic-latest
     runs-on: ubuntu-latest
+    env:
+      CONNECT_TOKEN: ${{ secrets.CONNECT_TOKEN }}
```

Comment:
why do we need this?

Reply:
i will upgrade this but wanted this to test the effects